src/go/rpk/pkg/cli/cmd/redpanda/config_test.go

```diff
@@ -254,25 +238,12 @@ redpanda:
           port: 9644
     developer_mode: true
 rpk:
-    enable_usage_stats: false
-    tune_network: false
-    tune_disk_scheduler: false
-    tune_disk_nomerges: false
-    tune_disk_write_cache: false
-    tune_disk_irq: false
-    tune_fstrim: false
-    tune_cpu: false
-    tune_aio_events: false
-    tune_clocksource: false
-    tune_swappiness: false
-    tune_transparent_hugepages: false
-    enable_memory_locking: false
-    tune_coredump: false
-    tune_ballast_file: false
-    overprovisioned: false
+    tune_network: true
+    tune_disk_scheduler: true
```

Comment:
Why are these true now?

Reply:
Since we are stripping all (because they are false) I just wanted the test to show that even if we strip the false, we keep the ones that are true.